include/pluginlib/impl/filesystem_helper.hpp

```diff
@@ -35,7 +35,19 @@
 #ifndef PLUGINLIB__IMPL__FILESYSTEM_HELPER_HPP_
 #define PLUGINLIB__IMPL__FILESYSTEM_HELPER_HPP_
 
-#if defined(__has_include)
+#if defined(_MSC_VER)
```

Comment:
I think it would make sense to compare the value to a minimal version of compiler as some Windows platforms with an older compiler will have the variable defined but not the filesystem header. I didnt track down the first version of MSVC implementing it but as we target VS2015 Update3 we should be safe by requiring `_MSC_VER >= 1900`